charts/values.yaml

```diff
@@ -28,6 +28,9 @@ group:
 ## The name of the host to join. If not provided, the deployment is a bootstrap host.
 bootstrapHostName: ""
 
+## Flag to enable to migrate from MarkLogic root to rootless image
+rootToRootlessUpgrade: false
```

Comment:
Using same parameter name _**rootToRootlessUpgrade**_ at line 32 and 48 for different pursposes may create confusion.
Instea add a sub-parameter named _**enabled**_ under rootToRootlessUpgrade for this flag.